Add links to the headerFilter and allowFiltering to the Header Filter description (#1090)
DevExpress/devextreme-demos: JSDemos/Demos/PivotGrid/HeaderFilter/description.md

> A [header filter](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/) allows a user to filter field values by including or excluding them from the applied filter. Click a header filter icon in the field chooser or on the field panel to open a popup menu that displays all unique field values.

Reviewer: It's not a good thing to lead the reader away from the demo right from the start. Besides, this doesn't convey the fact that the header filter is configured in the headerFilter object.

```suggestion
A header filter allows a user to filter field values by including or excluding them from the applied filter. Click a header filter icon in the field chooser or on the field panel to open a popup menu that displays all unique field values. To display header filter icons, enable the [allowFiltering](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowFiltering) property.
```

> Use the following properties to configure the header filter:

Reviewer: ```suggestion
To configure a header filter, use the global [headerFilter](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/) object or a field's [headerFilter](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/headerFilter/) object. This demo specifies the following properties in the global **headerFilter** object:
```

> - [allowFiltering](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowFiltering)        
> Allows users to toggle the checkbox in the popup window to filter fields. Set this property to **true** to display filter icons.
> 

Reviewer: ```suggestion
```
